projects/packages/status/src/class-status.php

```diff
@@ -240,7 +240,7 @@ public function is_staging_site() {
 		}
 
 		// Last, let's check if sync is erroring due to an IDC. If so, set the site to staging mode.
-		if ( ! $is_staging && method_exists( 'Jetpack', 'validate_sync_error_idc_option' ) && \Jetpack::validate_sync_error_idc_option() ) {
+		if ( ! $is_staging && method_exists( 'Automattic\\Jetpack\\Identity_Crisis', 'validate_sync_error_idc_option' ) && \Automattic\Jetpack\Identity_Crisis::validate_sync_error_idc_option() ) {
```

Comment:
Using `::class` can make it easier for tools to recognize it as a class reference.
```suggestion
		if ( ! $is_staging && method_exists( \Automattic\Jetpack\Identity_Crisis::class, 'validate_sync_error_idc_option' ) && \Automattic\Jetpack\Identity_Crisis::validate_sync_error_idc_option() ) {
```

Reply:
@anomiex but wouldn't this trigger an error if the class does not exist? which is precisely what we are testing there?

Reply:
well, not "precisely", but "one of the things"

Reply:
Yes this change would cause an issue and isn't something we want to make.

Reply:
> but wouldn't this trigger an error if the class does not exist?

No, it wouldn't. The `Foo::class` syntax is syntactic sugar for the corresponding string, but has the advantage that it's written out like any other class reference and follows `use` directives and the like as other class references do.

See https://3v4l.org/kEsCg for examples.